### PR TITLE
Add Hello World in Haskell

### PR DESCRIPTION
**Pull request by matheuscr30**

### CODE REVIEW

**Comment by matheuscr30** [comment]
@berviantoleo how i complete the travis file?

**Comment by berviantoleo** [comment]
simply put `make install-haskell` to `before_install`, and `make haskell ` to `script`. You must fill the command to that make file. `make haskell` should be how to run the script or code.

**Comment by matheuscr30** [comment]
I fix it, can you merge?

File: .travis.yml [hunk 1]
```diff
@@ -23,6 +24,7 @@ script:
   - make elixir
   - make fortran
   - make go
+  - make haskell
```
Comment (berviantoleo) [inline .travis.yml]: Missing `make haskell` in Makefile. You should add command what `make haskell` do.

Example in Makefile:

... [Another command]
haskell:
        [command to run haskell and the code]
... [Another command]

Looks like you updated how to install haskell.

**Comment by matheuscr30** [comment]
Sorry for the problems. I think now is OK

**Comment by berviantoleo** [comment]
Thanks

**Pull request merged**